identity governance: add examples of adding resources and resource roles in entitlement management (#13422)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/accesspackagecatalog-list-accesspackageresourceroles.md

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> Retrieve a list of [accessPackageResourceRole](../resources/accesspackageresourcerole.md) objects of an [accessPackageResource](../resources/accesspackageresource.md) in an [accessPackageCatalog](../resources/accesspackagecatalog.md). The resource would be one which was previously added to the catalog by [creating an accessPackageResourceRequest](accesspackageresourcerequest-post.md). This list of roles can then be used by the caller to select a role, which is needed when subsequently [creating an accessPackageResourceRoleScope](accesspackage-post-accesspackageresourcerolescopes.md).

Reviewer: ```suggestion
Retrieve a list of [accessPackageResourceRole](../resources/accesspackageresourcerole.md) objects of an [accessPackageResource](../resources/accesspackageresource.md) in an [accessPackageCatalog](../resources/accesspackagecatalog.md). The resource should have been added to the catalog by [creating an accessPackageResourceRequest](accesspackageresourcerequest-post.md). This list of roles can then be used by the caller to select a role, which is needed when subsequently [creating an accessPackageResourceRoleScope](accesspackage-post-accesspackageresourcerolescopes.md).
```

}
```

> ### Example 2: Retrieving the roles of a resource for a SharePoint Online site

Reviewer: ```suggestion
### Example 2: Retrieve the roles of a resource for a SharePoint Online site
```

#### Request

> The following is an example of the request.

Reviewer: Proposing that you explain this request further. I think it'll help give context to what you described in the description. Example: https://docs.microsoft.com/en-us/graph/api/accessreviewscheduledefinition-post?view=graph-rest-beta&tabs=http#example-2-create-an-access-review-on-all-teams-with-inactive-guest-users

Same for the response.

Author: Added more text to this example, thanks.
